extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -293,9 +293,8 @@ export class AmpLiveList extends AMP.BaseElement {
 
     if (hasNewInsert) {
       promise = promise.then(() => {
-        this.getVsync().mutate(() => {
-          // Should scroll into view be toggleable
-          this.viewport_./*OK*/scrollIntoView(this.element);
+        return this.getVsync().mutatePromise(() => {
+          this.viewport_.animateScrollIntoView(this.element);
```

Comment:
There's no need for mutate here, since animations create their own mutation context. You are certainly free to return promise from `animateScrollIntoView`, which in turn can simply do this by running `animate(...).then()`. You do not need to return `AnimatonPlayer` instance, unless you think halting animation is needed.